apps/dashboard/app/views/projects/_launcher_buttons.html.erb

```diff
@@ -6,53 +6,54 @@
 -%>
 
 <div class="row">
-
   <div class="col">
     <%= button_to(
       submit_project_launcher_path(@project.id, launcher.id),
-      class: 'btn btn-success',
+      class: 'btn btn-success launcher-button',
       title: 'Launch script with cached or default values',
       disabled: disabled,
       params: launcher.quick_launch_params,
       id: "launch_#{launcher.id}"
     ) do
     %>
-      <i class="fa fa-play"></i>
+      Launch
     <%- end %>
   </div>
 
   <div class="col">
     <%= link_to(
       project_launcher_path(@project.id, launcher.id),
-      class: "btn btn-success #{disabled_class}",
+      class: "btn btn-info launcher-button #{disabled_class}",
       id: "show_#{launcher.id}"
       ) do
     %>
-      <i class="fa fa-bars"></i>
+      Options
```

Comment:
Can we change this to maybe `View`? Not 100% sure about view, but don't quite like options either.

Reply:
I was trying to convey that a user can change the options passed with the job before launch is called.

- [x] View
- [ ] Options
- [ ] Settings
- [ ] Variables

View is good. I'll go with that unless any of the above sound better